test/Examples.hs

```diff
@@ -83,7 +83,8 @@ languages :: [LanguageExample]
 languages =
   [ le "python" ".py" "examples" (Just "script/known_failures.txt")
   , le "ruby" ".rb" "examples" (Just "script/known_failures.txt")
-  , le "typescript" ".ts" "examples" (Just "script/known_failures.txt")
+  , le "typescript" ".ts" "examples" (Just "typescript/script/known_failures.txt")
+  , le "typescript" ".tsx" "examples" (Just "typescript/script/known_failures.txt")
```

Comment:
I'm not familiar with this context, but is `"typescript"` as the language correct here rather than `"tsx"`?

Reply:
Yeah `"typescript"` refers to the directory the example repos are in. This adds about 150 tests to `parse-examples`.